### PR TITLE
Deploy to staging and production

### PR DESCRIPTION
**Pull request by rogeruiz**
Pairing with @rememberlenny

### CODE REVIEW

**Comment by rogeruiz** [comment]
This card _really_ represents the work / PR here => https://github.com/cloudfoundry-community/cf-uaa-guard-service/pull/6

**Comment by rogeruiz** [comment]
Should be working for staging now once 18F/cg-pipeline-tasks#41 is merged. It also has updated `environment_variables` introduced in cloudfoundry-community/cf-uaa-guard-service#6

**Comment by rogeruiz** [comment]
Going to hold off on deploying this in production until Monday since everyone is going to need to use Tock today to track their time. 😁 

We can still merge this in though. The `cg-deploy-uaa-guard-broker` pipeline is paused in Production.

**Review by LinuxBozo — APPROVED** [review]
LGTM :shipit:

**Comment by rememberlenny** [comment]
💯 💯 💯

File: pipeline.yml
Comment (rememberlenny) [inline pipeline.yml:12]: @rogeruiz Should we create in Favro a card to add these later?

Comment (rogeruiz) [reply pipeline.yml:12]: hmm, good point. I completely forgot about how we were moving away from GH issues and just putting everything in Favro. For now, let's just reference #2 in this card and if we get a chance to work on the assertion work we can. For now we'll focus on the PI9 plans we made `anti-fragility`. The base of this work for this PR was to make the `uaa-guard-service` deployment less fragile than it was before when it was artisanal and bespoke. Now that it's automated it's less fragile, but assertion tests would be good to have.

Comment (rogeruiz) [reply pipeline.yml:12]: @rememberlenny here's a good example. They shouldn't be too difficult to add. 

https://github.com/18F/cf-cdn-service-broker/blob/93cbea8487230b66a8258ff0572d26bcdcacbede/ci/acceptance-tests.sh

Comment (rememberlenny) [reply pipeline.yml:12]: @rogeruiz Got it!

**Pull request merged**
